### PR TITLE
Fix building extensions as WebAssembly modules.

### PR DESCRIPTION
**Pull request by PiotrSikora**
Signed-off-by: Piotr Sikora <piotrsikora@google.com>

### CODE REVIEW

**Comment by PiotrSikora** [comment]
Ping.

File: extensions/stats/plugin.cc [hunk 1]
```diff
@@ -36,6 +36,13 @@ namespace Plugin {
 
 namespace Stats {
 
+// TODO(PiotrSikora): move this back to Proxy-WASM SDK.
```
Comment (kyessenov) [inline extensions/stats/plugin.cc]: This got moved already. Care to update it to use Wasm::Common::TrafficDirection?

Comment (PiotrSikora) [reply extensions/stats/plugin.cc]: Done.

**Comment by PiotrSikora** [comment]
Yeah, and we need a presubmit as well, to catch build breaks... but I'm not sure if we can do docker-inside-docker for the `./build_wasm.sh` calls.

**Pull request merged**